pkg/generated-assets/generated_assets.go

```diff
@@ -0,0 +1,272 @@
+package generated_assets
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+	"gopkg.in/yaml.v2"
+	"k8s.io/klog/v2"
+)
+
+const (
+	manifestFileName              = "manifests.yaml"
+	ControllerDeploymentAssetName = "controller.yaml"
+	NodeDaemonSetAssetName        = "node.yaml"
+	MetricServiceAssetName        = "service.yaml"
+	MetricServiceMonitorAssetName = "servicemonitor.yaml"
+)
+
+// CSIDriverAssets contains all the assets required to deploy the CSI driver.
+// The assets are stored as raw bytes, and can be retrieved using GetAsset.
+// The assets can be saved to a directory using Save and loaded using NewFromAssets function.
```

Comment:
The comment doesn't help to understand who and when is supposed to call NewFromAssets function.

Reply:
I added few comments around, here, to `Save()` and to `NewFromAssets()`